packages/shared/ReactErrorUtils.js

```diff
@@ -105,6 +105,32 @@ const ReactErrorUtils = {
   },
 };
 
+if (__DEV__) {
+  (ReactErrorUtils: any)._suppressedErrorsInDEV =
+    typeof WeakSet === 'function' ? new WeakSet() : null;
+
+  (ReactErrorUtils: any).markErrorAsSuppressedInDEV = function(error: any) {
+    const suppressedErrors = (ReactErrorUtils: any)._suppressedErrorsInDEV;
```

Comment:
Will an expando no longer work? `error._suppressLogging === true`

Reply:
If we use an expando, then I think you can get rid of the extra ReactErrorUtils methods.

Reply:
You could throw a frozen object though.

Reply:
I guess it's uncommon enough that I can try/catch around it?

Reply:
Yeah seems fine